taxi/models.py

```diff
@@ -15,7 +16,15 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(
+        max_length=8, unique=True, validators=[
+            RegexValidator(
+                regex=r"^[A-Z]{3}\d{5}$",
+                message="License number must have next format -> 'ABC12345'",
+                code="invalid_license_number"
```

Comment:
Move these checks to forms since you wont be able to create custom license number, for example, via admin panel.